Source/MonoGame.Extended.Content.Pipeline/MonoGame.Extended.Content.Pipeline.csproj

```diff
@@ -97,7 +97,7 @@
     <Compile Include="Tiled\TmxTerrain.cs" />
     <Compile Include="Tiled\TmxTile.cs" />
     <Compile Include="Tiled\TmxTileOffset.cs" />
-    <Compile Include="Tiled\TmxTileset.cs" />
+    <Compile Include="Tiled\TmxTileSet.cs" />
```

Comment:
I think [tileset is one word](https://www.google.com.au/search?q=tileset) so it should have a lowercase s. Was this stopping the project building on Linux?